src/ProportionalLiquidity.sol

```diff
@@ -234,11 +234,28 @@ library ProportionalLiquidity {
         address account,
         uint256 amount
     ) private {
-        curve.totalSupply = mintAdd(curve.totalSupply, amount);
-
-        curve.balances[account] = mintAdd(curve.balances[account], amount);
-
-        emit Transfer(address(0), msg.sender, amount);
+        require(amount > 1000, "Proportional Liquidity/amount too small!");
```

Comment:
Just wondering, do we need this test every time some adds LP or only during the first deposit (i.e., curve.totalSupply == 0)?

I see from your test that the small second deposit works fine so just asking since I'm confused. I figured once the initial LP is added, any arbitrary amount of LP (even very small) could be added.

Otherwise, LGTM

Reply:
aha, good catch
updated to require to be greater than 1000 only for the first deposit